<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Faculty</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Field</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Index</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_Name"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_lName"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_faculty"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_field"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_Index">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_sortuj">
          <property name="text">
           <string>find student </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_wyswietl">
          <property name="text">
           <string>show all students</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_usun">
          <property name="text">
           <string>delete student</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_dodaj">
          <property name="text">
           <string>add student</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_import">
          <property name="text">
           <string>import from xml file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>save to xml file</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
